### PR TITLE
Add Jenkins job to build an fpm package

### PR DESCRIPTION
**Pull request by alexmuller**
This runs a build script in the packager repo which will make a Debian package.

### CODE REVIEW

File: modules/govuk_jenkins/templates/jobs/build_fpm_package.yaml.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+- scm:
+    name: build_fpm_package
```
Comment (surminus) [inline modules/govuk_jenkins/templates/jobs/build_fpm_package.yaml.erb]: In other jobs the scm name and job name are different, I'm not sure if this is a requirement or just an oddity?

**Comment by surminus** [comment]
Other than my comment (which is incredibly minor), looks good, merge away :+1:

**Comment by deanwilson** [comment]
Looks good. Merge away.

File: modules/govuk_jenkins/templates/jobs/build_fpm_package.yaml.erb (hunk 1 above)
Comment (alexmuller) [inline modules/govuk_jenkins/templates/jobs/build_fpm_package.yaml.erb]: I think that's just the way that we've done stuff in the past but I think we should probably refactor the whole way we do SCM for Jenkins jobs at some point.

**Pull request merged**